nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/core/ElasticSearchAdminImpl.java

```diff
@@ -549,4 +555,20 @@ public boolean isReady() {
     public List<String> getInitializedRepositories() {
         return repositoryInitialized;
     }
+
+    @Override
+    public Optional<ESHintQueryBuilder> getHintByOperator(String name) {
+        return Optional.ofNullable(hints.get(name));
+    }
+
+    /**
+     * Sets the {@link #hints} from the hint descriptors
+     * 
+     * @since 11.1
```

Comment:
This is not in `11.1`, is it?

Reply:
i agree ;) the good version is `10.10-HF17`